specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```diff
@@ -3380,6 +3400,244 @@
+    },
+    "ConnectionMonitorHTTPConfiguration": {
+      "properties": {
+        "method": {
+          "type": "string",
+          "description": "HTTP method.",
+          "enum": [
+            "Get",
+            "Post"
+          ],
+          "x-ms-enum": {
+            "name": "HTTPConfigurationMethod",
+            "modelAsString": true
+          }
+        },
+        "port": {
+          "type": "integer",
+          "description": "The destination port on which a HTTP test is to be performed."
+        },
+        "path": {
+          "type": "string",
+          "description": "The path to add to the provided URI."
+        },
+        "requestHeaders": {
+          "type": "array",
+          "description": "The HTTP request headers.",
+          "items": {
+            "$ref": "#/definitions/HTTPHeader"
+          }
+        },
+        "validStatusCodeRanges": {
+          "type": "array",
+          "description": "A list of acceptable HTTP status code ranges. For instance: [ \"100\", \"200-205\", \"3xx\" ]",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "description": "Describes a configuration for a HTTP test."
+    },
+    "ConnectionMonitorSuccessThreshold": {
+      "properties": {
+        "checksFailedPercent": {
+          "type": "integer",
+          "description": "Percentage of failed checks."
+        },
+        "roundTripTimeMs": {
+          "type": "number",
+          "description": "Maximum roundtrip time in milliseconds."
+        }
+      },
+      "description": "Describes the threshold at which a connection monitor test configuration is considered successful."
+    },
+    "ConnectionMonitorEndpointFilter": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Include"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectionMonitorEndpointFilterType",
+            "modelAsString": true
+          },
+          "description": "The type of connection monitor endpoint filter. The accepted value is \"Include\"."
+        },
+        "items": {
+          "type": "array",
+          "description": "A list of connection monitor endpoint items that define this filter.",
+          "items": {
+            "$ref": "#/definitions/ConnectionMonitorEndpointItem"
+          }
+        }
+      },
+      "required": [
+        "items"
+      ],
+      "description": "Describes a connection monitor endpoint filter."
+    },
+    "ConnectionMonitorEndpointItem": {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "AgentAddress"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectionMonitorEndpointItemType",
+            "modelAsString": true
+          },
+          "description": "The type of the connection monitor endpoint item. The accepted value is \"AgentAddress\"."
```

Comment:
> The accepted value is \"AgentAddress\ [](start = 75, length = 38)

nit - you may remove this. may get missed, when updating enum values in the future